feature_tests/kt-conf.toml

```diff
@@ -0,0 +1,2 @@
+domain = "dev.gigapixel"
+lib_name = "somelib"
```

Comment:
I would prefer that this is picked up from the tool because it's user to load the dylib, and it would be convenient if it were the same as the generate lib file. But I couldn't figure it out